bundle-workflow/src/test_workflow/config/test_manifest.yml

```diff
@@ -12,4 +12,56 @@ components:
       test-configs:
         - with-security
         - without-security
+  - name: anomaly-detection
+    integ-test:
+      dependencies:
+        - job-scheduler
+        - alerting
```

Comment:
`job-scheduler` and `alerting` won't be dependencies for all plugins. Since we are not using this object yet, may be you could keep empty for all other plugins or add the correct dependencies for each plugin to avoid confusion.

Reply:
I think we should just remove them to keep things simple https://github.com/opensearch-project/opensearch-build/pull/512#discussion_r710581526. The dependency management logic needs to be handled via maven eventually.